src/v/kafka/protocol/errors.h

```diff
@@ -218,6 +218,8 @@ enum class error_code : int16_t {
     // The broker rejected this static consumer since another consumer with the
     // same group.instance.id has registered with a different member.id.
     fenced_instance_id = 82,
+    // Can not find/run inline wasm transformation.
+    inline_wasm_error = 83,
```

Comment:
This is not valid. The list of errors returned from Kafka are defined as a part of the protocol: https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/common/protocol/Errors.java

83 in particular is already taken by ELIGIBLE_LEADERS_NOT_AVAILABLE.

This is one instance where we are constrained by the Kafka protocol. I think the best we can do is maybe use CORRUPT_MESSAGE and include a tag for messages to clients, but clients will not know about the tag unless they are specifically looking. In particular, librdkafka will not know what to do with the tag.

This also reminds me that we should write a KIP to reserve a tag range for proxies / third party Kafka implementations to use. cc @senior7515 @dotnwat